dat/outfits/lib/multicore.lua

```diff
@@ -113,19 +205,115 @@ function multicore.init( params )
             sec="#y"..p_("core","secondary").."#n",
          }).."#0"
       end
+
+      local averaged = is_engine( po ) and is_multiengine( p )
+      local multicore_off = p and po and marked_n( p, po:id() )
+      local sm = smfs.readdir( p, {engines_comb_dir} )
+      local total = smfs.readdir( p, {engines_comb_dir.."_total"} )
+      local totaleml = (total and total["engine_limit"]) or 0
+
+      if averaged then
+         if multicore_off~=true then
+            desc = desc .. fmt.f(_("\n#oLoad Factor:#0 #y{share}%#0 #o({eml} {t} out of {total} {t} )#0"),{
+               share = (sm[po:id()] and sm[po:id()]["part"]) or 0,
+               eml = (sm[po:id()] and sm[po:id()]["engine_limit"]) or 0,
+               total = totaleml,
+               t = eml_unit
+            })
+         end
+      end
+
       for k,s in ipairs(stats) do
-         desc = desc.."\n"..add_desc( s, nomain, nosec )
+         local off = multicore_off and (nosec or nomain) and s.name~="mass"
+         desc = desc.."\n"..add_desc( s, nomain or off, nosec or off )
+         if averaged and s.name == "engine_limit" and multicore_off~=true then
+            desc = desc .. fmt.f(_("\n\t#o[#0 #y{share}%#0 #oout of {total} {t}#0 #o]#0"),{
+               share = sm[po:id()] and sm[po:id()]['part'] or 0,
+               total = totaleml,
+               t = eml_unit
+            })
+         end
+      end
+
+      if multicore_off ~= nil then
+         desc = desc .. "\n#bWorking Status: #0"
+         if multicore_off == false then
+            desc = desc .. "#grunning#0"
+         else
+            desc = desc .. "#rHALTED#0"
+         end
```

Comment:
```suggestion
         local status
         if multicore_off == false then
            status = desc .. "#g".._("running").."#0"
         else
            status = desc .. "#r".._("HALTED").."#0"
         end
         desc = desc .. "\n#b"..fmt.f(_("Working Status: {status}"), {status=status})
```
Should be easier to translate this way.

Reply:
Promise, one day I will stop forgetting this.

Reply:
Fixed it in some edits. I really hate doing changes in the github interface. It doesn't even use a monospace font...

Reply:
So you can't even indent properly ?
So unpractical..